java/query/query-core/src/main/java/sleeper/query/model/Query.java

```diff
@@ -33,12 +33,12 @@ public class Query {
 
     private Query(Builder builder) {
         processingConfig = Objects.requireNonNull(builder.processingConfig, "processingConfig must not be null");
```

Comment:
Please can this validation be moved to the builder as well? It looks like it might be a mistake that this uses Objects.requireNonNull, and it should probably use validateNonNull?